chartfx-chart/src/test/java/de/gsi/chart/plugins/EditAxisTests.java

```diff
@@ -0,0 +1,150 @@
+package de.gsi.chart.plugins;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.concurrent.CountDownLatch;
+
+import javax.swing.SwingUtilities;
+
+import javafx.application.Platform;
+import javafx.embed.swing.JFXPanel;
+import javafx.util.Duration;
+
+import org.junit.jupiter.api.Test;
+
+import de.gsi.chart.XYChart;
+import de.gsi.chart.axes.AxisMode;
+import de.gsi.chart.axes.spi.DefaultNumericAxis;
+import de.gsi.chart.plugins.EditAxis.AxisEditor;
+
+/**
+ * Basic interface tests for EditAxis plugin
+ *
+ * @author rstein
+ */
+public class EditAxisTests {
+    private boolean jfxIsSetup;
+
+    @Test
+    public void basicEditAxisInterfaceTests() {
+
+        assertDoesNotThrow(() -> new EditAxis());
+
+        assertDoesNotThrow(() -> {
+            EditAxis plugin = new EditAxis(false);
+            assertEquals(false, plugin.isAnimated());
+
+            plugin.setAnimated(true);
+            assertEquals(true, plugin.isAnimated());
+
+            plugin.setZoomDuration(Duration.millis(100));
+            assertEquals(Duration.millis(100), plugin.getZoomDuration());
+
+            plugin.setZoomDuration(Duration.millis(120));
+            assertEquals(Duration.millis(120), plugin.getZoomDuration());
+
+            assertEquals(0, plugin.popUpList.size());
+            plugin.removeMouseEventHandlers(null);
+            assertEquals(0, plugin.popUpList.size());
+        });
+
+        assertDoesNotThrow(() -> {
+            for (AxisMode mode : AxisMode.values()) {
+                EditAxis plugin = new EditAxis(mode);
+                assertEquals(mode, plugin.getAxisMode());
+            }
+        });
+
+    }
+
+    @Test
+    public void attachChartTests() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=4989692)